### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Aleks-ars**
- [DEMO LINK](https://Aleks-ars.github.io/layout_stars/)
- [TEST REPORT LINK](https://Aleks-ars.github.io/layout_stars/report/html_report/)

- [ x ] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [ x ] Each BEM block has its own separate file
- [ x ] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [ x ] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,42 @@
...
+}
+
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 14px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  margin: 0;
+  margin-top: 2px;
```
Comment (loralevitska) [inline src/style.css:17]: Combine with `margin` shorthand property, fix all cases if exist

File: src/style.css [hunk 2]
```diff
@@ -1 +1,42 @@
...
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 14px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
```
Comment (loralevitska) [inline src/style.css:15]: Combine with `background ` shorthand property, fix all cases if exist

File: src/style.css [hunk 3]
```diff
@@ -1 +1,42 @@
...
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
```
Comment (loralevitska) [inline src/style.css:40]:
```suggestion
.stars--5 .stars__star,
.stars--4 .stars__star:nth-child(-n+4),
.stars--3 .stars__star:nth-child(-n+3),
.stars--2 .stars__star:nth-child(-n+2),
.stars--1 .stars__star:first-child {
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Redeploy your solution and check links before re-request review
image

**Comment by Aleks-ars** [comment]
Errors have been corrected

**Pull request closed without merging**